docs/runbooks/CnaoDown.md

```diff
@@ -2,29 +2,31 @@
 
 ## Meaning
 
-Cluster-network-addons-operator (CNAO) deploys additional networking components on top of the cluster.
-This alert fires when CNAO operator is down.
+The `Cluster-network-addons-operator` (CNAO) deploys additional networking components on top of the cluster.
+This alert fires when CNAO is down.
 
 ## Impact
 
-With CNAO operator down, changes in the components are not reconciled, and as a result might not be properly deployed.
+With CNAO down, the cluster cannot reconcile changes to virtual machine components. As a result, the changes might fail to take effect.
 
 ## Diagnosis
 
 - Check CNAO's operator pod namespace:
```

Comment:
Suggestion: "Obtain the namespace of the CNAO pod:"

No need to say "operator" because CNAO includes the word.